frontend/src/components/Health/HealthIndicator.tsx

```diff
@@ -2,26 +2,24 @@ import * as React from 'react';
 import { PopoverPosition, Tooltip } from '@patternfly/react-core';
 import { HealthDetails } from './HealthDetails';
 import * as H from '../../types/Health';
-import { Size, createIcon } from './Helper';
-import { createTooltipIcon } from '../../config/KialiIcon';
+import { createIcon, createTooltipIcon } from '../../config/KialiIcon';
 import { healthIndicatorStyle } from '../../styles/HealthStyle';
 
 interface HealthIndicatorProps {
   health?: H.Health;
   id: string;
-  size?: Size;
```

Comment:
Health indicator size should be aligned wiith current text next to indicator (font-size). For this reason I have removed specific size property.